flowauth/backend/flowauth/login.py

```diff
@@ -22,7 +22,7 @@ def signin():
     user = User.query.filter(User.username == json["username"]).first()
     if user is not None:
         if user.is_correct_password(json["password"]):
-            login_user(user)
+            login_user(user, remember=False)
```

Comment:
What's the purpose of this change? Isn't `False` the [default](https://flask-login.readthedocs.io/en/latest/#flask_login.login_user) anyway?